src/explore-education-statistics-admin/src/pages/admin-dashboard/components/ManagePublicationsAndReleasesTab.tsx

```diff
@@ -267,7 +270,19 @@ const ManagePublicationsAndReleasesTab = ({
             {apiErrors.length > 0 ? (
               <ErrorSummary id="publications-error" errors={apiErrors} />
             ) : (
-              <LoadingSpinner />
+              <>
+                {loading ? (
+                  <LoadingSpinner />
+                ) : (
+                  <p>
```

Comment:
Would wrap this with `<LoadingSpinner loading={loading}>`

Reply:
Still haven't done this. `LoadingSpinner` takes children and can wrap other elements.

Reply:
Done

Reply:
Done